cumulusci/cumulusci.yml

```diff
@@ -51,18 +51,6 @@ tasks:
             managed: True
             namespace_token: '%%%NAMESPACE%%%'
             filename_token: '___NAMESPACE___'
-    dx_push:
-        description: Uses Salesforce DX to push code to a scratch workspace org
-        class_path: cumulusci.tasks.salesforcedx.BaseSalesforceDXTask
-        options:
-            command: force:src:push
-            options: '-a'
-    filter_subscribers:
```

Comment:
Why are filter_subscribers and generate_apex_docs deleted? Seems they should be left in.